Source/Atomic/Resource/ResourceCache.cpp

```diff
@@ -64,6 +64,8 @@ static const char* checkDirs[] =
 };
 
 static const SharedPtr<Resource> noResource;
+const char* Atomic::PAK_EXTENSION = ".pak";
+const char* Atomic::PAK_FILTER = "*.pak";
 

```

Comment:
Ok, Android doesn't like the Atomic:: here, as already in the Atomic namespace.  I'll land this and then fix it up in post :)